types/node/child_process.d.ts

```diff
@@ -4,17 +4,18 @@ declare module "child_process" {
     import * as net from "net";
 
     interface ChildProcess extends events.EventEmitter {
-        stdin: stream.Writable;
-        stdout: stream.Readable;
-        stderr: stream.Readable;
-        stdio: [stream.Writable, stream.Readable, stream.Readable];
-        killed: boolean;
-        pid: number;
+        stdin: stream.Writable | null;
+        stdout: stream.Readable | null;
+        stderr: stream.Readable | null;
+        readonly channel: boolean;
+        readonly stdio: Array<stream.Writable | null>;
```

Comment:
Why the change of `stdio`? I think [1] and [2] should be kept as `stream.Readable | null`.

Reply:
Oh yea, this is correct, is there a way to declare the first few elements as one type and the `rest` as others?

Reply:
I can do `        readonly stdio: [stream.Writable | null, stream.Readable | null, stream.Readable | null, ...Array<stream.Readable | stream.Writable | null>];
` for TS 3.x, but not for TS 2.x

Reply:
Don't know if this is possible with <3.x.
You could just add one or two `stream.Readable | stream.Writable | null` elements which is not fully correct but will most likely fit for most users. Seems noone missed them till now anyway...